bio_market_store/store/static/css/add_product_style.css

```diff
@@ -1,16 +1,10 @@
-body {
-    background-color: #f8f9fa;
-    font-family: Arial, sans-serif;
-    text-align: center;
-}
-
 h1 {
     color: #343a40;
     margin-bottom: 20px;
 }
 
 form {
```

Comment:
There are a lot of overwrites because of using `form {}` and related to all forms in the projects.
If you could set an additional class for example (add-product-form), it will change only your form.

The same with others.

Reply:
![Screenshot 2025-02-21 234202](https://github.com/user-attachments/assets/bf0e6e71-406d-4cd5-9ab0-f3e5967beed3)